### PR TITLE
Add Corretto21-AL2023

### PR DESCRIPTION
**Pull request by a1flecke**
- Add image containing Corretto 21 and AL2023
- There was no corretto:21-al2023-generic variant so I chose corretto:21-al2023 per the guidance [on the dockerhub Corretto page](https://hub.docker.com/_/amazoncorretto) under Image Variants

### CODE REVIEW

**Comment by rmacconn** [comment]
This is awesome.  I've been needing this image.

**Comment by spelfrey** [comment]
This will be very helpful. Thank you!

**Comment by danny-garrett** [comment]
Looking forward to this!

**Comment by joeyweisband** [comment]
This is huge, would love to have this ASAP

**Comment by admendoz25** [comment]
Good stuff @a1flecke!

File: README.md [hunk 1]
```diff
@@ -214,6 +215,7 @@ Some come from the parent images and some are installed in this image for backwa
 | amazoncorretto-20-al2023      |     | ✔️    | ✔️   | ✔️    | ✔️     | ✔️    |        | ✔️   |     |
 | amazoncorretto-20-debian      |     |      | ✔️   | ✔️    | ✔️     | ✔️    |        |     |     |
 | amazoncorretto-21             |     | ✔️    | ✔️   | ✔️    | ✔️     | ✔️    |        | ✔️   |     |
+| amazoncorretto-20-al2023      |     | ✔️    | ✔️   | ✔️    | ✔️     | ✔️    |        | ✔️   |     |
```
Comment (KeltonKarboviak) [inline README.md:218]:
```suggestion
| amazoncorretto-21-al2023      |     | ✔️    | ✔️   | ✔️    | ✔️     | ✔️    |        | ✔️   |     |
```

**Pull request merged**